src/css/product-menu.css

```
@@ -187,6 +187,20 @@
     text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.3);
 }
 
+.tag-filter {
+    display: block;
+    width: 100%;
+    max-width: 300px;
+    margin: 20px auto;
+    padding: 10px;
+    border-radius: 5px;
+    border: 1px solid #ccc;
+    background-color: #fff;
+    font-size: 1rem;
+    color: #333;
+    transition: border-color 0.3s ease;
+}
+
 /* Responsive adjustments */
 @media (max-width: 768px) {
```

```suggestion
@media (width <= 768px) {
```

Wist je dat je je media queries ook zo kunt schrijven?: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries#targeting_media_features